docs/get-started/getting-started.mdx

```diff
@@ -29,6 +29,58 @@ ds library style sheet in your entry point, in this example we use React (App.js
 
 <br />
 <br />
+
+## 💅 Customize theme
+
+If you want to use a provider to configure the library's theme, you can do so using the ThemeProvider provided by dd360-ds. This will allow you to set your own settings for the theme.
```

Comment:
I think it should not really be optional to use the provider, otherwise it won't even load our default palette.

```suggestion
To use the library it's important that you wrap the application on our `Provider`. You can will customize with your own palette of colours and pass to our provider without complexity
```

Reply:
It is not really that important to use the provider, the application does not break because it has a default theme in the library